AppIntegrity/README.md

```diff
@@ -0,0 +1,53 @@
+#  App Integrity
+
+## Package description
+
+This package contains the logic to implement app integrity checks using [Firebase App Check](https://firebase.google.com/docs/app-check). It therefore encorporates the Firebase SDK as a dependency, along with the Networking module to enable backend requests.
+
+## Overview
+
+### What is app integrity?
+
+App Integrity describes checks made (in this case, by Firebase AppCheck) to verify that a genuine instance of your app is accessing your backend resources and that the device accessing it has not been tampered with.
+
+### Why do we need it for mobile?
+
+As a public client of STS, our service is susceptible to malicious attacks from unauthorized clients. We have therefore implemented app integrity checks when the `/token` endpoint is called to ensure that:
```

Comment:
I think this is slightly the wrong way around:
I think is STS supports public clients and because the client is public (i.e. not running on a server we control) we need a way to ensure that the client calling the service is a genuine (signed) instance of our app.

Reply:
I'm not sure I see a difference between the two 😭
What exactly is the client in this situation? Is it the app?